cmake/bob_robotics.cmake

```diff
@@ -138,20 +140,23 @@ macro(BoB_project)
         list(APPEND PARSED_ARGS_BOB_MODULES robots/gazebo)
     endif()
 
+    # We always need the common module so that main() is defined
+    list(APPEND PARSED_ARGS_BOB_MODULES common)
+
     # Do linking etc.
     BoB_build()
 
     # When using the ARSDK (Parrot Bebop SDK) the rpath is not correctly set on
     # Ubuntu (and presumably when linking against other libs in non-standard
     # locations too). This linker flag fixes the problem.
-    if(UNIX)
+    if(UNIX AND NOT APPLE)
         set(CMAKE_EXE_LINKER_FLAGS -Wl,--disable-new-dtags)
     endif()
 
     # Copy all DLLs over from vcpkg dir. We don't necessarily need all of them,
     # but it would be a hassle to figure out which ones we need.
     if(WIN32)
-        file(GLOB dll_files "$ENV{VCPKG_ROOT}/installed/${CMAKE_GENERATOR_PLATFORM}-windows/bin/*.dll")
+        file(GLOB dll_files "${VCPKG_PACKAGE_DIR}/bin/*.dll")
         foreach(file IN LISTS dll_files)
             get_filename_component(filename "${file}" NAME)
             if(NOT EXISTS "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${filename}")
```

Comment:
I think this is a tiny bit dubious as it won't copy if vpkg updates  the library so you might end up with a dll/header conflict. I **think** cmake provides a "copy if different" mechanism http://cmake.3232098.n2.nabble.com/copy-if-different-on-build-td7583855.html

Reply:
That's a good shout; thanks. It is currently a bit of a hack.